queue_services/entity-filer/tests/unit/filing_processors/test_incorporation_filing.py

```diff
@@ -0,0 +1,42 @@
+# Copyright © 2020 Province of British Columbia
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""The Unit Tests for the Incorporation filing."""
+
+import copy
+from unittest.mock import patch
+
+from entity_filer.filing_processors import incorporation_filing
+from tests.unit import INCORP_FILING, create_business, create_filing
```

Comment:
Not sure if this matters too much or not, but it might be a better idea to import the incorp filing from the example data in the schema repo (in case the schema changes - that way we don't have to change our test data in the schema repo and this one). That's what I did for parties

Reply:
Thats a great idea. Will make the change.